data/profile.d/rocky.conf

```diff
@@ -0,0 +1,24 @@
+# Anaconda configuration file for Rocky Linux.
```

Comment:
Please, update also the `data/profile.d/Makefile.am` file. That should fix the failing tests.

Reply:
Hello - I added a rocky.conf line to `Makefile.am` as suggested. Let me know if you need anything else!